WikipediaUnitTests/Code/SectionEditorFindAndReplaceTests.swift

```diff
@@ -274,7 +275,8 @@ class SectionEditorFindAndReplaceTests: XCTestCase {
         webView.evaluateJavaScript("""
                 var line = editor.state.search.posFrom.line
                 var ch = editor.state.search.posFrom.ch
-            window.webkit.messageHandlers.\(messageHandlerKeyNextSearchLocation).postMessage({'\(cursorLineKey)': line, '\(cursorCharacterKey)': ch})
+                window.webkit.messageHandlers.\(messageHandlerKeyNextSearchLocation).postMessage({'\(cursorLineKey)': line, '\(cursorCharacterKey)': ch})
+                true
```

Comment:
Are these `true`s necessary or were they just tinkering code?

Reply:
Oddly enough they seem to be necessary on iOS 14 - I fount this issue while running the tests locally against iOS 14.